### PR TITLE
Flag --show-all has been deprecated

### PR DESCRIPTION
**Pull request by mofelee**

### CODE REVIEW

File: content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Note that the job name and pod name are different.
 
 ```shell
 # Replace "hello-4111706356" with the job name in your system
-$ pods=$(kubectl get pods --show-all --selector=job-name=hello-4111706356 --output=jsonpath={.items..metadata.name})
+$ pods=$(kubectl get pods --all-namespaces --selector=job-name=hello-4111706356 --output=jsonpath={.items..metadata.name})
```
Comment (xiangpengzhao) [inline content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md]: Actually, just removing `--show-all` would be OK here. You don't need to use `--all-namespaces` instead in this context.
/cc @deads2k

Comment (bradtopol) [reply content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md]: I agree with @xiangpengzhao Reading the deprecation notes the proper correction is to just remove the --show-all from the kubectl command.

**Comment by bradtopol** [comment]
See my inline comment for need revisions to this PR. Thanks for the contribution

**Comment by mofelee** [comment]
@xiangpengzhao @bradtopol deleted `--show-all` flag

**Comment by xiangpengzhao** [comment]
/lgtm
Defer to @bradtopol  for approval :)

**Pull request merged**